src/main/java/ch/njol/skript/classes/data/SkriptClasses.java

```diff
@@ -658,6 +659,9 @@ public void change(final Slot[] slots, final @Nullable Object[] deltas, final Ch
 					@Override
 					@Nullable
 					public Color parse(String input, ParseContext context) {
+						if (input.contains("RED:")) {
```

Comment:
Maybe you could add a helper function for testing if ``input`` is a ColorRGB String in ColorRGB itself?

Reply:
Good thinking. Updated 👍